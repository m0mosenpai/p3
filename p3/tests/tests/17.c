#include <stdio.h>
#include <unistd.h>

int main(){
    char *buf = "hello worldhello worldhello world";
    fprintf(stdout, "hello world!\n");
    fflush(stdout);
    fprintf(stderr, "this is err\n");
    fflush(stderr);
    return 0;
}