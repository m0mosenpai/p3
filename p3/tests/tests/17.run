timeout 30 ../solution/wsh tests/17.wsh
