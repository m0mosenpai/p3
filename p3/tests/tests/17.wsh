grep hello <tests/16.c
gcc ./tests/17.c -o ./tests/17.tmp
./tests/17.tmp &>tests/17-out
echo hi >>tests/17-out
./tests/17.tmp &>>tests/17-out
echo hiagain >>tests/17-out
diff tests/17-out tests/17.out.orig
exit