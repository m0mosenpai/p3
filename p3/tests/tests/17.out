    char *buf = "hello worldhello worldhello world";
    printf("hello world!\n");
